src/Shared/LayoutRenderers/IISInstanceNameLayoutRenderer.cs

```diff
@@ -29,31 +28,16 @@ namespace NLog.Web.LayoutRenderers
     [LayoutRenderer("iis-site-name")]
     // ReSharper disable once InconsistentNaming
     [ThreadAgnostic]
-    public class IISInstanceNameLayoutRenderer : LayoutRenderer
+    public class IISInstanceNameLayoutRenderer : AspNetHostEnvironmentLayoutRendererBase
```

Comment:
Think the complexity of "inheriting" the ability to lookup HostEnvironment outweights the code-use.

Reply:
OK, that can be adjusted

Reply:
removed common base class